content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -160,6 +160,9 @@ The following table shows which referential action each database supports.
 | MongoDB††   | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **❌**     |
 
 - † On MySQL, `SetDefault` is ignored and is silently replaced with `NoAction` instead.
+  On MySQL 8.0+, when a `SetDefault` referential action is triggered, a foreign key violation error is raised.  
+  On older versions, like MySQL 5.6+, creating or altering a table with a `SetDefault` referential action is not possible at all.
+  We therefore advise against using `SetDefault` at all on MySQL.
```

Comment:
See earlier comment thread: https://github.com/prisma/docs/pull/4011#discussion_r1015765067

Reply:
How about:

> While MySQL 8.0+ accepts table definitions with the `SET DEFAULT` referential action (i.e., `SetDefault` in a Prisma schema definition), this action is actually not supported by InnoDB - MySQL's default storage engine and the only one Prisma supports. In fact, InnoDB silently interprets `SET DEFAULT` as `NO ACTION`. This is evident when an update or deletion triggers a `SetDefault` referential action, as InnoDB would raise the same foreign key violation error you would see with `NoAction`.
> On older versions, like MySQL 5.6+, creating or altering a table with a `SetDefault` referential action is not possible at all, as it would fail with a syntax error when attempting to push the Prisma schema to the database.
> We therefore advise against using `SetDefault` at all on MySQL.

Reply:
We might want to figure out with @keerlu if there is a better way to go about this. 

We need to list `SetDefault` as working in the table - but we also need to explain people that it does not _actually_ work properly. So maybe ✔️/❌? 

Also, having this full explanation in a footnote seems suboptimal. Maybe another section somewhere that explains these "weird" cases where we do not just want to put " ✔️ " in the table?